acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ProcessPantheonNodeRunner.java

```diff
@@ -75,17 +74,6 @@ public void startNode(final PantheonNode node) {
     params.add("--bootnodes");
     params.add(String.join(",", node.bootnodes().toString()));
 
-    final PermissioningConfiguration permissioningConfiguration =
-        node.getPermissioningConfiguration();
-    if (permissioningConfiguration.isNodeWhitelistSet()) {
-      params.add("--nodes-whitelist");
-      params.add(String.join(",", permissioningConfiguration.getNodeWhitelist().toString()));
-    }
-    if (permissioningConfiguration.isAccountWhitelistSet()) {
-      params.add("--accounts-whitelist");
-      params.add(String.join(",", permissioningConfiguration.getAccountWhitelist().toString()));
-    }
-
```

Comment:
Now that we have --permissions-nodes-enabled and --permissions-accounts-enabled, we need to set those here instead